package.json

```diff
@@ -40,6 +40,7 @@
   "devDependencies": {
     "browserify": "^13.0.0",
     "d3": "3.5",
+    "lodash.result": "^4.4.0",
```

Comment:
I think the fact that this will be called at runtime (and not just in the tests) makes this a `dependency` and not just a `devDependency`. That would also be my biggest concern, given that crossfilter doesn't currently have runtime dependencies. Otherwise, looks pretty cool!